src/cc/lib/graph/graph.cc

```diff
@@ -125,9 +125,14 @@ void Graph::GetNodeFeature(std::span<const NodeId> node_ids, std::span<snark::Fe
         }
```

Comment:
Please, add a test with a request to node features for:
1. Node is not present in the graph. Expected dense feature vector should be 0.
2. Node is present in the graph, but feature is not present in any of partitions. FV should be 0s as well.